docs/oauth2.md

```diff
@@ -0,0 +1,30 @@
+Oauth2
+======
+
+Oauth2 is a django app that will can be used to cache your `django-oauth-toolkit`
+access token model.
```

Comment:
Change made. Removed as a dev-requirement and added to the project `extra_require`